<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect_PS</class>
 <widget class="QWidget" name="connect_PS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Настройки подключения пикамперметра:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>COM порт:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbPort"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Настройки подключения сервера:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_IP_adress"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_port"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Настройки сохранения:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_TXT">
         <property name="text">
          <string>формат TXT</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_CSV">
         <property name="text">
          <string>формат CSV</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_BIN">
         <property name="text">
          <string>формат BIN</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_Close_window">
         <property name="text">
          <string>Отменить изменения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Connect">
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
